lessons/26-js-functions/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+function showEvenNumbers (lowerLimit, upperLimit) {
+    for (let i=lowerLimit; i<=upperLimit; i++){
+        if (i%2 === 0) {
+            console.log (i);
+        }
+    }
+}
+
+showEvenNumbers (0, 100);
+
+function showUser (name, age, phoneNumber, street, postalCode, isMarried) {
+    console.log (` Name: ${name} 
+               \n Age: ${age}
+               \n Phone Number: ${phoneNumber}
+               \n Street: ${street}
+               \n Postal Code: ${postalCode}`);
```

Comment:
When you use back-ticks to create a string you don't need to use `\n`, you can simple just hit enter for a new line like you already have 😄